keps/sig-architecture/2019-03-19-go-modules.md

```diff
@@ -166,9 +213,14 @@ This section is in progress, pending investigation into necessity and feasibilit
 
 ### Graduation Criteria
 
-* kubernetes/kubernetes vendor management uses go modules
-* published staging components can be successfully consumed by go module consumers
-* published staging components can be successfully consumed by GOPATH-based consumers
+* `k8s.io/kubernetes` vendor management uses go modules
+* there are documented processes for:
+  * adding/pinning a new dependency
+  * updating the pinned version of an existing dependency
+  * removing an unnecessary dependency
+* published staging components can be successfully consumed by:
+  * go module consumers using `go get`
+  * GOPATH-based consumers using `go get`
```

Comment:
so we accept breaking glide and dep?

Reply:
this explicitly calls for a Godeps.json for staging modules, so we shouldn't break anyone who isnt broken today?

Reply:
> this explicitly calls for a Godeps.json for staging modules, so we shouldn't break anyone who isnt broken today?

correct, the intent is to keep giving tools that consume Godeps.json data a best-effort representation of our dependencies